apps/dashboard/src/components/workflow-editor/steps/email/configure-email-step-preview.tsx

```diff
@@ -0,0 +1,79 @@
+import * as Sentry from '@sentry/react';
+import { HTMLAttributes, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { useStep } from '@/components/workflow-editor/steps/step-provider';
+import { usePreviewStep } from '@/hooks';
+import { EmailPreviewHeader } from '@/components/workflow-editor/steps/email/email-preview';
+import { Separator } from '@/components/primitives/separator';
+import { Skeleton } from '@/components/primitives/skeleton';
+import { ChannelTypeEnum } from '@novu/shared';
+import { cn } from '@/utils/ui';
+
+type SmallEmailPreviewProps = HTMLAttributes<HTMLDivElement>;
+const SmallEmailPreview = (props: SmallEmailPreviewProps) => {
```

Comment:
If not mistaken we call this Mini preview.

```suggestion
const MiniEmailPreview = (props: SmallEmailPreviewProps) => {
```